canteenWeb/tests.py

```diff
@@ -63,3 +72,134 @@ def test_order_reject(self):
 
     def tearDown(self):
         Order.objects.all().delete()
+
+    # TODO: Complete tests.
+    # Vikrant's note: Order rejection and acceptance has been tested in Postman. Going on a vacation so will complete tests later.
+
+
+class MenuTest(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.URL = "menu_item"
+
+        # Basic user.
+        cls.user = User.objects.create(
+            username="Student1", email="sixtynine@one.com", password="super-secret-meow"
+        )
+        cls.user.is_staff = True
+        cls.user.save()
+
+        # Client.
+        cls.client = APIClient()
+        cls.client.force_authenticate(user=cls.user)
+
+    def setUp(self):
+        # Create an menu before all tests.
+        self.sandwich = MenuItem.objects.create(
+            name="Sandwich",
+            price=random.randint(0, 100),
+            is_available=True,
+            options="JAIN",
```

Comment:
You forgot a field which is required.

Before creating sandwich, create

```python

self.indian = Category.objects.create(
    name="Indian"
)

self.sandwich = MenuItem.object.create(
    ... # Whatever you have
    category=self.indian
)

```

Reply:
Added it.